ftl/python/benchmark/main.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+from ftl.common import args
+from ftl.benchmark import benchmark
+
+
+_RUNTIME = "python"
+
+parser = args.base_parser()
+python_parser = argparse.ArgumentParser(
```

Comment:
These benchmark flags are repeated in each benchmark main, it might be worth putting these in something like benchmark/args.py so they are not repeated for each language.

Reply:
good idea, i made the change!